bihire/Broadcatser
# #169580229 Add a red-flag/intervention

### What does this Pr do ?
A user should be able to add a red-flag or intervention page
### Description of the task to be completed ?
User should be able to choose between red-flag or intervention while creating a item
User should be able to have the following fields while on the create page
- title
- type  // [red-flag, intervention]
-  location // Lat Long coordinates
-  images  [Image, Image],
-  videos  [Image, Image],
-  comment

User should have a fixed navbar

### How should this be manually tested?
after cloning this repository
- go to UI directory
- open add_item.html in chrome you should be able to see the mentioned above page

### Any background context you want to provide?
N/A
### What are the relevant pivot tracker stories ?
#169580229
https://www.pivotaltracker.com/story/show/169580229

#### screenshots if available?
![image](https://user-images.githubusercontent.com/37307172/68406165-ff15fd80-0189-11ea-9a78-c28b44cb9ec2.png)
![image](https://user-images.githubusercontent.com/37307172/68406209-13f29100-018a-11ea-8c77-0cbbfb93619d.png)
